packages/es-components/src/components/controls/buttons/Button.js

```diff
@@ -9,9 +9,14 @@ const StyledButton = styled.button`
   background: none;
   border: none;
   box-sizing: border-box;
-  display: ${props => (props.block ? 'block' : 'inline-block')};
+  display: block;
   padding: 0;
-  width: ${props => (props.block ? '100%' : 'initial')};
+  width: 100%;
+
+  @media (min-width: ${props => props.theme.screenSize.tablet}) {
```

Comment:
This will default to `block` on phone and tablet sizes, and will only respect the prop on desktop. Is that right?

Reply:
yes, that is correct. :)